src/main/java/com/hivemq/cli/commands/AbstractCommonFlags.java

```diff
@@ -324,5 +329,16 @@ public void setCleanStart(final @Nullable Boolean cleanStart) {
         this.cleanStart = cleanStart;
     }
 
+    @Nullable
+    public MqttWebSocketConfig getWebSocketConfig() {
+        if (useWebsockets) {
+            return MqttWebSocketConfig.builder()
+                    .serverPath("/mqtt")
```

Comment:
Although we are using this path on broker.hivemq.com, should this be really the default?

Reply:
Hmm, i thought this is the standard path for mqtt over websockets.

What do you think about adding a `-ws:path` option to specify the path and using `/mqtt` via default properties?

Reply:
yes, this is good idea

Reply:
We should probably also support passing a single url like
`ws://broker.hivemq.com:8000/mqtt` or `wss://broker.hivemq.com:8001`